cmake/jerry.cmake

```diff
@@ -83,6 +83,13 @@ if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
   list(APPEND DEPS_LIB_JERRY_ARGS -DFEATURE_ERROR_MESSAGES=ON)
 endif()
 
+# Nuttx is not using the default port implementation of JerryScript
+if("${TARGET_OS}" MATCHES "NUTTX")
+  list(APPEND DEPS_LIB_JERRY_ARGS -DJERRY_PORT_DEFAULT=OFF)
```

Comment:
Good question. Unfortunately I'm not familiar with TizenRT. CC: @yichoi, @glistening